operations.adoc

```diff
@@ -6,6 +6,32 @@ Changes to this document must be approved by a simple majority vote of the Jakar
 
 This is a public document.
 
+== Specification Committee
+
+The Specification Committee is responsible for implementing the Eclipse Foundation Specification Process (EFSP) for all Specification Projects (as that term is defined by the EFSP). This includes the adaptation of the EFSP for Jakarta which is referred to as the Jakarta EE Specification Process (JESP). 
```

Comment:
This position has been in the back of mind for quite some time...  Why does the Jakarta EE Spec Committee define the EFSP for all of the projects within the Eclipse Foundation?  I understand why we did that originally.  But, now that it is "complete", it seems that the EFSP should be governed by an independent group outside of the Jakarta EE working group.